### PR TITLE
README updated

### PR DESCRIPTION
**Pull request by j-tirelli**

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -18,33 +19,65 @@ const generateRecords = function(numRecords) {
...
-            console.log('example data file has been written');
-          }
-          mongoose.disconnect();
-        });
-      });
+const seedController = (args) => {
+  let numOfRecords = args[0] || 700;
+  let chunkSize = args[1] || numOfRecords;
+  let callback = callBackGenerator(args[2]);
+  let chunks = numOfRecords / chunkSize;
```
Comment (luke-eastman) [inline database/seed.js]: could chunks end up being a fraction here? maybe that wouldn't matter anyway.

File: database/seed.js [hunk 2]
```diff
@@ -18,33 +19,65 @@ const generateRecords = function(numRecords) {
...
+
+const callBackGenerator = (option = 'fs') => {
+  if (option === 'fs') {
+    return (records) => {
+      fs.writeFile('./data/example_data.js', `module.exports = ${JSON.stringify(records)}`, (err) => {
+        if (err) {
+          console.log('error:', err);
+        } else {
+          console.log('example data file has been written');
+          debugger;
```
Comment (luke-eastman) [inline database/seed.js]: probably don't want the debugger in here

**Review by luke-eastman — APPROVED** [review]
Looks good! There are a few console logs sprinkled around but some of those are nice to have in a data generation script.

File: database/seed.js (hunk 1 above)
Comment (j-tirelli) [reply database/seed.js]: Good Catch! I actually forgot to remove this file from my commit as it is going through a heavy revision now as well...

**Pull request merged**